packages/vkui/src/components/CarouselBase/CarouselBase.tsx

```diff
@@ -77,30 +91,12 @@ export const CarouselBase = ({
 
   const isCenterAlign = align === 'center';
 
-  /*
-   * Считает отступ слоя галереи во время драга
-   * Используется только для looped=false галереи
-   * так как только у нее есть пределы по краям
-   */
-  const calculateDragIndent = () => {
-    const localMax = slidesManager.current.max ?? 0;
-    const localMin = slidesManager.current.min ?? 0;
-    const indent = shiftXCurrentRef.current + shiftXDeltaRef.current;
-
-    if (indent > localMax) {
-      return localMax + Number((indent - localMax) / 3);
-    } else if (indent < localMin) {
-      return localMin + Number((indent - localMin) / 3);
-    }
-
-    return indent;
-  };
-
   const calculateCanSlideLeft = () => {
     if (looped) {
       return !slidesManager.current.isFullyVisible;
     }
-    return !slidesManager.current.isFullyVisible && shiftXCurrentRef.current < 0;
+    const isStartShiftX = isBiggerOrEqual(shiftXCurrentRef.current, 0, isRtl);
```

Comment:
```suggestion
    const isStartShiftX = isLowerOrEqual(shiftXCurrentRef.current, 0, isRtl);
```
тут разве не так? было ведь `shiftXCurrentRef.current < 0`

Reply:
Сейчас все работает корректно, сейчас сложно сказать, скорее всего логика где-то поменялось и пришлось тут условие поменять. Если использовать isLowerOrEqual то не работает